### PR TITLE
rsync: fix IPv6 on musl

### PR DESCRIPTION
**Pull request by paper42**
close: #26796

### CODE REVIEW

File: srcpkgs/rsync/template [hunk 1]
```diff
@@ -10,11 +10,14 @@ makedepends="zlib-devel acl-devel popt-devel xxHash-devel liblz4-devel
...
 license="GPL-3.0-only"
-homepage="http://samba.anu.edu.au/rsync/"
+homepage="https://rsync.samba.org/"
 changelog="https://download.samba.org/pub/rsync/NEWS"
-distfiles="http://www.samba.org/ftp/rsync/src/rsync-$version.tar.gz"
+distfiles="https://www.samba.org/ftp/rsync/src/rsync-$version.tar.gz"
 checksum=becc3c504ceea499f4167a260040ccf4d9f2ef9499ad5683c179a697146ce50e
 
+# Force enable IPv6 on musl - upstream bug https://bugzilla.samba.org/show_bug.cgi?id=10715
+CFLAGS="$CFLAGS -DINET6"
```
Comment (ericonr) [inline srcpkgs/rsync/template:19]: You can set `CFLAGS="-DINET6"` directly.

**Review by ericonr — COMMENTED** [review]
Wow, that bug is 6 years old at this point :/

**Comment by pullmoll** [comment]
Ouch. I never actually tested whether rsync supports / uses IPv6 despite using it everywhere and all the time.

File: srcpkgs/rsync/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/rsync/template]: thanks

**Comment by ericonr** [comment]
Thanks!

**Pull request merged**